src/stripe.android.ts

```diff
@@ -1,6 +1,7 @@
 import * as utils from 'tns-core-modules/utils/utils';
 import { android as androidApp } from "tns-core-modules/application";
-import { CardBrand, CardCommon, CreditCardViewBase, PaymentMethodCommon, StripePaymentIntentCommon, StripePaymentIntentStatus, Token } from './stripe.common';
+import { CardBrand, CardCommon, CreditCardViewBase, PaymentMethodCommon, StripePaymentIntentCommon, StripePaymentIntentStatus, Token, Source } from './stripe.common';
```

Comment:
Imports get ordered alphabetically by the `from` source, and the names within `{}` are also alphabetized.

What that means here is move `import *`... to line 2 (this one may have been wrong in the original source), and put `Source` before `StripePaymentIntentCommon`.